internal/vsp/feepayment.go

```diff
@@ -630,6 +630,66 @@ func (c *Client) status(ctx context.Context, ticketHash *chainhash.Hash) (*ticke
 	return &resp, nil
 }
 
+func (c *Client) setVoteStatus(ctx context.Context, ticketHash *chainhash.Hash, choices []wallet.AgendaChoice) error {
+	w := c.Wallet
+	params := w.ChainParams()
+
+	ticketTx, err := c.tx(ctx, ticketHash)
+	if err != nil {
+		return fmt.Errorf("failed to retrieve ticket %v: %w", ticketHash, err)
+	}
+	if len(ticketTx.TxOut) != 3 {
+		return fmt.Errorf("ticket %v has multiple commitments: %w", ticketHash, errNotSolo)
+	}
+
+	if !stake.IsSStx(ticketTx) {
+		return fmt.Errorf("%v is not a ticket", ticketHash)
+	}
```

Comment:
let's reverse these two checks, so if it does not have 3 outputs and is not a ticket, we don't report it as one.